apisix/core/config_etcd.lua

```diff
@@ -48,14 +49,15 @@ local mt = {
         return " etcd key: " .. self.key
     end
 }
+local etcd_version = _M.local_conf().etcd.version or "v2"
```

Comment:
Create a new `config_etcd_v3.lua` for v3, instead of mixing the logic of v2 and v3, it may be more clear.

Reply:
I thought config_etcd_v3.lua would reuse quite a lot code of config_etcd.lua, it might result in difficulty in maintenance.